### PR TITLE
init add of config sensor tf module

### PR DESCRIPTION
**Pull request by retr0h**
currently supports the Azure cloud enrichment service.

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  description = "(optional) the azure storage account where enrichment data is stored"
+  type        = string
+  default     = ""
+}
+
+variable "enrichment_storage_container_name" {
+  description = "(optional) the container where enrichment data is stored"
+  type        = string
+  default     = ""
+}
```
Comment (retr0h) [inline variables.tf:72]: Maybe this is a generic name across enrichment and not specific to Azure?

File: cloud-config/init.tpl [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+    owner: root:root
+    path: /etc/corelight/corelightctl.yaml
+    permissions: '0644'
+
+runcmd:
+  - corelightctl sensor bootstrap -v
+  - corelightctl sensor deploy -v
+
+%{ if cloud_provider == "azure" ~}
```
Comment (retr0h) [inline cloud-config/init.tpl:37]: Didn't really get too fancy here.

**Comment by retr0h** [comment]
Can see usage in https://github.com/corelight/terraform-config-sensor/pull/1

**Review by thathaneydude — APPROVED** [review]
Looks good! Looking forward to migrating the AWS and Azure modules to use this.

**Review by thathaneydude — APPROVED** [review]

**Pull request merged**